README.md

```diff
@@ -67,6 +67,22 @@ data = o3.get_multiple_city_air([ARRAY OF CITY NAMES])
 
 ![Gif of ozone.get_multiple_city_air()](/src/media/ozone_get_multiple_city_air_updated.gif)
 
+### Ozone Parameters
+
+Ozone can fetch the following parameters:
+
+ * aqi: air quality index, a measurement of air quality that tells you how clean or polluted the air is
```

Comment:
I would recommend changing aqi into `aqi` to make it more unambiguous that it's a code argument. Same for other parameters as well.